src/golioth_coap_client.c

```diff
@@ -239,6 +239,7 @@ static void nack_handler(
     }
 }
 
+#ifndef GOLIOTH_DONT_OVERRIDE_LIBCOAP_LOG_HANDLER
```

Comment:
Is there a reason to use "double negative" here? In my head, I have to translate "if not not override" to "override", so it might be easier to do something like:

```c
#ifdef GOLIOTH_OVERRIDE_LIBCOAP_LOG_HANDLER
```

Reply:
Updated.